desktop/apps/auction2/client.js

```diff
@@ -44,6 +45,15 @@ const calendar = new AddToCalendar({
   el: $('.auction2-callout')
 })
 
+// render the associated auction if there is one
+if (auction.get('associated_sale')) {
+  const associatedSale = auction.get('associated_sale')
+  mountAuctionBlock({
+    relatedAuction: true,
+    sale: associatedSale
```

Comment:
Updated this to accept a `sale` instead of all of the individual attributes as props!